logagg_master/cli.py

```diff
@@ -0,0 +1,417 @@
+
+        if ping_result['result']['success']:
+            if ping_result['result']['details'] == 'Authentication passed':
+                master_details = {'host': host, 'port': port, 'key': auth.key, 'secret': auth.secret, 'admin': True}
+                self.state['master'] = master_details
+                self.state.flush()
+                prGreen('Added master with admin permission')
+            elif ping_result['result']['details'] == 'Authentication failed' and not auth.key and not auth.secret:
+                master_details = {'host': host, 'port': port, 'key': auth.key, 'secret': auth.secret, 'admin': False}
+                self.state['master'] = master_details
+                self.state.flush()
+                prYellow('Added master with non-admin permission')
+            else:
+                err_msg = ping_result['result']['details']
+                prRed(err_msg)
+
+
+    def list_master(self):
+        '''
+        Show Master details
+        '''
+        master = self.ensure_master()
+        headers = ['HOST', 'PORT', 'ADMIN']
+
+        data = [[master.host, master.port, str(master.admin)]]
+        print(tabulate(data, headers=headers))
+
+
+    def add_nsq(self,  nsqd_tcp_address, nsqd_http_address):
+        '''
+        Add nsq details to master
+        '''
+        master = self.ensure_master()
+
+        if not master.admin:
+            err_msg = 'Requires admin permissions to master'
+            prRed(err_msg)
+            sys.exit(0)
+
+        add_nsq_url = self.MASTER_ADD_NSQ_URL.format(host=master.host,
+                                                        port=master.port,
+                                                        nsqd_tcp_address=nsqd_tcp_address,
+                                                        nsqd_http_address=nsqd_http_address,
+                                                        key=master.key,
+                                                        secret=master.secret)
+
+        add_nsq_result = self.request_master_url(add_nsq_url)
+
+        if add_nsq_result['result']['success']:
+            prGreen(add_nsq_result['result']['details'])
+        else:
+            err_msg =  add_nsq_result['result']['details']
+            prRed(err_msg)
+
+
+    def list_nsq(self):
+        '''
+        List nsq details of master
+        '''
+        master = self.ensure_master()
+
+        if not master.admin:
+            err_msg = 'Requires admin permissions to master'
+            prRed(err_msg)
+            sys.exit(0)
+
+        get_nsq_url = self.MASTER_GET_NSQ_URL.format(host=master.host,
+                                                     port=master.port,
+                                                     key=master.key,
+                                                     secret=master.secret)
+
+        get_nsq_result = self.request_master_url(get_nsq_url)
+    
+        if get_nsq_result['result']['success']:
```

Comment:
please do use something like below
```
get_nsq_result.get('result', {}).get('success', False)
```
and also please do change in all other places